src/Filament/Resources/UserResource.php

```diff
@@ -72,6 +73,9 @@ public static function table(Table $table): Table
                 Tables\Columns\TextColumn::make('email_verified_at')
                     ->label(__('cachet::user.list.headers.email_verified_at'))
                     ->dateTime(),
+
+                Tables\Columns\ToggleColumn::make('is_admin')
+                    ->label(__('cachet::user.list.headers.is_admin')),
```

Comment:
```suggestion
                    ->label(__('cachet::user.list.headers.is_admin_label')),
```

Reply:
@jbrooksuk Are you sure? I was trying to stick to the naming convention for tables, adding `_label` seems to be the convention used for forms? 🤔

Reply:
Ah yeah, you're right.